Doc/tutorial/introduction.rst

```diff
@@ -478,74 +478,103 @@ example::
-  are evaluated  from the left to the right.
-
-* The :keyword:`while` loop executes as long as the condition (here: ``a < 10``)
-  remains true.  In Python, like in C, any non-zero integer value is true; zero is
-  false.  The condition may also be a string or list value, in fact any sequence;
-  anything with a non-zero length is true, empty sequences are false.  The test
-  used in the example is a simple comparison.  The standard comparison operators
-  are written the same as in C: ``<`` (less than), ``>`` (greater than), ``==``
-  (equal to), ``<=`` (less than or equal to), ``>=`` (greater than or equal to)
-  and ``!=`` (not equal to).
-
-* The *body* of the loop is *indented*: indentation is Python's way of grouping
-  statements.  At the interactive prompt, you have to type a tab or space(s) for
-  each indented line.  In practice you will prepare more complicated input
-  for Python with a text editor; all decent text editors have an auto-indent
-  facility.  When a compound statement is entered interactively, it must be
-  followed by a blank line to indicate completion (since the parser cannot
-  guess when you have typed the last line).  Note that each line within a basic
-  block must be indented by the same amount.
-
-* The :func:`print` function writes the value of the argument(s) it is given.
-  It differs from just writing the expression you want to write (as we did
-  earlier in the calculator examples) in the way it handles multiple arguments,
-  floating point quantities, and strings.  Strings are printed without quotes,
-  and a space is inserted between items, so you can format things nicely, like
-  this::
-
-     >>> i = 256*256
-     >>> print('The value of i is', i)
-     The value of i is 65536
-
-  The keyword argument *end* can be used to avoid the newline after the output,
-  or end the output with a different string::
+   4
+
+Function argumets
+-----------------
+
+We already know :func:`print` function, that writes the value of the
+argument(s) it is given on screen.  The arguments are specified inside
+parentheses ``()``. In simplest form, f.e. ``print(a, b)`` the arguments
+are positional.  That means function process them in same order we wrote
+them.::
+
+   >>> group = "Knights"
+   >>> say_what = '"Ni!"'
+   >>> print(group, 'Who Say', say_what)
+   Knights Who Say "Ni!"
+
+The keyword argument *end* can be used to avoid the newline after the output,
+or end the output with a different string::
+
+   >>> count = 0
+   >>> count = 10
+   >>> while count > 5:
+   ...     print(count, end=' ')
+   ...     count = count - 1
+   ...
+   10 9 8 7 6
+
+Fibonacci series
+----------------
+
+We now know all we need to write code that will print all numbers from the
+Fibonacci series! Let's check what nubers in it are lower than 1000.
 
      >>> a, b = 0, 1
      >>> while a < 1000:
-     ...     print(a, end=',')
+     ...     print(a, end=', ')
```

Comment:
Do we want the space in `end` in this example? If so, update the output on line 546 with spaces after commas, so doctest passes on the CI.

Reply:
Thank you for your feedback.  I did this to showcase a different output style compared to the previous example, which used one character. My hope it might support a desire to experiment further with the end keyword. Maybe we might come up with a better way to do it. Or just not mess with it here 😂.

Reply:
In 1fae12b I tried a bit more obvious "two characters" and concepts from another example to create maybe a more interesting final example. Let me know what you think.